maestro-test/src/test/resources/062_copy_paste_text.yaml

```diff
@@ -0,0 +1,5 @@
+appId: com.example.app
+---
+- copyText:
```

Comment:
Let's call it `copyTextFrom` to not create an illusion that it will copy from a currently focused view